### ISSUE TITLE
[SCREENREADER]: Text Appointments - Mobile Phone group should have aria-live and aria-atomic attributes for context

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2433]
## Description

The message that we are updating a user's mobile phone preferences and the success message are not being announced to assistive device users. I think we should include all of these sub-sections in the main container and add `aria-live` and `aria-atomic` regions to the container. Screenshot attached.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, I want to hear the update message when my preferences are being saved.
* As a screen reader user, I want to hear the success message read when my preferences are saved.

## Possible Fixes (optional)

I think we can add `aria-live` and `aria-atomic` attributes to the containing DIV, and move the checkbox and update message into the parent container. I've created a [public Codepen](https://codepen.io/tpierce_402/pen/JjjXJXp) to illustrate the different ways live regions are handled.

```html
<div
  aria-atomic="false"
  aria-live="polite"
  class="vet360-profile-field"
  data-field-name="mobilePhone"
>
  ...
</div>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-14 at 3 33 09 PM](https://user-images.githubusercontent.com/934879/66781526-220a0600-ee99-11e9-8bda-4e7615cdbd50.png)

**Comment by lkoenigsberg** [comment]
@elwoodva , this one I think we should do as part of the mobile text preference release.  Please let me know if the complexity is huge!

**Comment by elwoodva** [comment]
@1Copenut In issue 2581 we are going to make a change so that there is no checkbox here on the profile page.  It's only going to be updated through the edit modal.  How does that affect the changes you are suggesting here on this issue?  Also, all of the phone numbers work exactly the same way and share the code.  So would I be able to assume that this should not be changed just for the mobile phone?

**Comment by lkoenigsberg** [comment]
Note that we will still be having checkboxes on the profile page and make appropriate copy updates to support the 508 issue at https://github.com/department-of-veterans-affairs/va.gov-team/issues/2428.
